docs/index.md

```diff
@@ -28,6 +28,12 @@ python main.py \
     video_paths="[./sample/v_ZNVhz7ctTq0.mp4, ./sample/v_GGSY1Qvo990.mp4]"
 
 # use `device_ids="[0, 2]"` to run on 0th and 2nd devices in parallel
+
```

Comment:
Sorry, this chunk does not belong to the quick start. Could you remove it?

However, please add this possibility to the comment: 
"use `device_ids="[0, 2]"` to run on 0th and 2nd devices in parallel"
->
"use `device_ids="[0, 2]"` to run on 0th and 2nd devices in parallel or add `cpu=true` " to use CPU